File: week3/misc/scores.c
#include <stdio.h>
#include <cs50.h>

//Prototype
float average(int length, int array[]);

int main(void){

    const int TOTAL = 3;

    // Want the values for all three scores, input by the user

    int scores[TOTAL];
    for (int i = 0; i < TOTAL; i++){

        scores[i] = get_int("Scores");
    }

    // Function average(below) is calculated, then used in printf

    printf("The average is %f\n", average(TOTAL, scores));

}

float average(int length, int array[])
{

    // calculate the average, which is the sum of all values, divided by number of values

    int sum = 0;

    for(int i = 0; i < length; i++)
    {
    sum = sum + array[i];
    }
    return sum/ (float) length;

}

File: week6/lecture/scores.py
from cs50 import get_int

# scores = [72, 73, 33]

# print("Average: " + str((sum(scores) / len(scores))))

# print(f"Average:  {sum(scores) / len(scores)} ")

# average = (sum(scores) / len(scores))
# print(f"Average : {average}")

# print()
# print()
# ################################################
# print("Another way of doing the scores codes")

scores = []

for i in range(3):
    scores.append(get_int("Enter your three scores: "))

average = (sum(scores) / len(scores))
print(f"Average: {average}")